src/openrct2/ride/TrackData.cpp

```diff
@@ -6435,6 +6535,10 @@ static constexpr const rct_string_id RideConfigurationStringIds[] = {
     STR_EMPTY,                         // 264
     STR_EMPTY,                         // 265
     STR_EMPTY,                         // 266
+    STR_LARGE_HALF_LOOP_LEFT,
+    STR_LARGE_HALF_LOOP_RIGHT,
+    STR_LARGE_HALF_LOOP_RIGHT,
+    STR_LARGE_HALF_LOOP_LEFT,
 };
```

Comment:
Sort out the formatting of the file to align and use the same convention for comments as the other entries

Reply:
I've given them the TrackElemType names instead of the TrackElemType index, I think the index is a holdover from when the file was in openrct2-ui.